chart/templates/crds/pipeline.yaml

```diff
@@ -0,0 +1,93 @@
+apiVersion: apiextensions.k8s.io/v1
+kind: CustomResourceDefinition
+metadata:
+  name: pipelines.captain.benthos.dev
+  labels:
+  {{- include "benthos-captain.labels" . | nindent 4 }}
```

Comment:
cant we use .Values.labels ?

Reply:
I just generated this chart but it looks like it does a few things in the `_helpers.tpl` file that we can't do in the values file, such as access `AppVersion`

```yaml
{{- define "benthos-captain.labels" -}}
helm.sh/chart: {{ include "benthos-captain.chart" . }}
{{ include "benthos-captain.selectorLabels" . }}
{{- if .Chart.AppVersion }}
app.kubernetes.io/version: {{ .Chart.AppVersion | quote }}
{{- end }}
app.kubernetes.io/managed-by: {{ .Release.Service }}
{{- end }}
```